packages/services/src/view-service/delegations-by-address-index.ts

```diff
@@ -107,12 +107,6 @@ export const delegationsByAddressIndex: Impl['delegationsByAddressIndex'] = asyn
               },
               metadata: denomMetadata ? customizeSymbol(new Metadata(denomMetadata)) : undefined,
               extendedMetadata,
-              equivalentValues: [
-                {
-                  equivalentAmount: { hi: 0n, lo: 0n },
-                  numeraire: STAKING_TOKEN_METADATA,
-                },
-              ],
```

Comment:
I could add a registry dependency for `view-service`, but I don't see much point in doing that just for the sake of adding zero equivalentValues

Reply:
Hmm... this means, though, that the staking page will not show the equivalent value of `0 UM` for delegation tokens that the user _doesn't_ hold, which is needed as part of the staking page design. If we're pulling metadata from IndexedDB, we don't need to hit the registry — we could just query for the staking token metadata in our database.

Reply:
In order to get metadata from indexed-db I need to know which metadata corresponds to the staking token. And at the moment, I can only get the assetId of staking token from registry. And this leads to the fact that I have to access the registry at some point.

Reply:
Ahh, true. Hm. We do know that the `display` will always be `penumbra` though, right? So we could look for the metadata that has that property?

Or, we could hard-code the UM asset ID in code, since that should also never change, right?

Reply:
Yes, it is. 
But at the same time, Gabe suggested adding `STAKING_TOKEN` to the registry and now, it seems to me, we should consider the registry the only reliable source about `STAKING_TOKEN`

(actually, now I think this is quite redundant, and this PR would be much less if we had left STAKING_TOKEN just a constant)

Reply:
> we should provide assetid of staking token to view service as a HandlerContext item

Do I understand correctly that in this case we will pass the assetid of staking token as context key for all rpc calls, although it is used only in one? 
I have my doubts about this way in general, as we may end up with a lot of context keys, which will always be passed but rarely used

Reply:
In order to pass stakingTokenId as a context key you need to do a lot of work. 
The thing is that service-worker cannot access registry because it has no chain-id, we get chain-id when initializing services through querier, and for that querier must be initialized.   
I don't think it shouldn't be done, but I suggest to do it in a separate PR on HanlderContext refactoring 

How will the design change in this PR:
<img width="300" alt="image" src="https://github.com/penumbra-zone/web/assets/25391690/6253d7ea-11b6-4d09-8f28-b82df3b945aa">
<img width="300" alt="image" src="https://github.com/penumbra-zone/web/assets/25391690/0e076787-c6eb-4742-a294-7ebb46e59c87">